docs/001_develop/03_client-capabilities/004_header/index.mdx

```diff
@@ -8,4 +8,219 @@ tags:
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration:
+```html
+  <foundation-header></foundation-header>
+```
+
+Usage:
+```typescript
+@customElement({
+  name: 'header-example',
+  template: html`
+    <foundation-header
+      :routeNavItems=${(x) => x.navItems}
+    ></foundation-header>
+  `,
+})
+export class HeaderExample extends GenesisElement {
+  const navItems = [{
+    title: "Home",
+    icon: {
+      name: "home",
+      variant: "solid",
+    },
+    navId: "header",
+    placementIndex: 0,
+    routePath: "home",
+    routeName: "home",
+  },
+  {
+    title: "Grids",
+    icon: {
+        name: "table",
+        variant: "solid",
+      },
+    navId: "header",
+    placementIndex: 1,
+    routePath: "grids",
+    routeName: "grids",
+  }];
+}
+```
+
+</TabItem>
+<TabItem value="react">
+
+Declaration:
+```html
+  <foundation-header></foundation-header>
+```
+
+Usage:
+```typescript
+export default function HeaderExample({}) {
+  const navItems = [{
+    title: "Home",
+    icon: {
+      name: "home",
+      variant: "solid",
+    },
+    navId: "header",
+    placementIndex: 0,
+    routePath: "home",
+    routeName: "home",
+  },
+  {
+    title: "Grids",
+    icon: {
+        name: "table",
+        variant: "solid",
+      },
+    navId: "header",
+    placementIndex: 1,
+    routePath: "grids",
+    routeName: "grids",
+  }];
+
+  return (
+    <foundation-header
+      routeNavItems={navItems}
+    ></foundation-header>
+  );
+}
+```
+
+</TabItem>
+<TabItem value="angular">
+
+Declaration:
+```html
+  <foundation-header></foundation-header>
+```
+
+Usage
+```typescript
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+@Component({
+  selector: 'header-example',
+  template: `
+    <foundation-header
+      [routeNavItems]="navItems"
+    ></foundation-header>
+  `,
+  standalone: true,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+})
+export class HeaderExampleComponent {
+  navItems = [{
+    title: "Home",
+    icon: {
+      name: "home",
+      variant: "solid",
+    },
+    navId: "header",
+    placementIndex: 0,
+    routePath: "home",
+    routeName: "home",
+  },
+  {
+    title: "Grids",
+    icon: {
+        name: "table",
+        variant: "solid",
+      },
+    navId: "header",
+    placementIndex: 1,
+    routePath: "grids",
+    routeName: "grids",
+  }];
+}
+```
+</TabItem>
+</Tabs>
+
+<PrimitiveHeaderExample />
+
+### Attributes
+
+| Attribute | Type | Use | Example |
+|---|---|---|---|
+| logo-src | `any` | Optional attribute which sets the source of the image in the navigation bar and flyout menu. The Genesis logo will be shown if this attribute is not provided. | `<foundation-header logo-src="path/to/logo.png">` |
```

Comment:
Hmm, I also think it should be typed as a string. Right now, it’s set to any because it is defined as any in the foundation-header:
![image](https://github.com/user-attachments/assets/b5eb6f30-f07e-49ce-a1ca-452d13a2c713)

I’ll correct it here to a string , and then I’ll also submit a fix for foundation-ui